Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.6-slim
```

Comment:
We're using python buildpack when deploying to PaaS. Docker container is currently only used for running tests on Jenkins, similar to what notifications-api and admin do (but we build a new container each time instead of mapping the workspace as a volume, which makes the process a bit simpler)